docs/tutorials/projects/verification-system.md

```diff
@@ -18,31 +17,48 @@ no_list: true
 # SMEs: Bijan Haney
 ---
 
-In the [registry](https://app.viam.com/module/viam-labs/verification-system), Viam Labs provides a model of [vision service classifier](/ml/vision/classification/) called `verification-system`.
+In the [registry](https://app.viam.com/module/viam-labs/verification-system), Viam Labs provides a [vision service classifier](/ml/vision/classification/) model called `verification-system`.
 Follow this tutorial to learn how to configure this module and set up your own security system to detect, verify, and alarm based on specified detections from nested [vision service detectors](/ml/vision/detection/).
 
 You can use whatever combination of detectors and camera you would like with this module, but follow along with our recommendations to layer an [`mlmodel` person detector](#configure-an-mlmodel-person-detector) and [facial detector](#configure-a-facial-detector) together, creating a security system with facial recognition.
+With this combination of detectors layered together, your verification system will trigger a countdown when it detects people and disarm the alarm if it detects your face with the facial detector within the countdown period.
```

Comment:
So this should come first and you should explain more what this project does. Speaking of "whatever combination of detectors" may just about work for someone familiar with what a detector is and how they work but we should provide some help for forming a mental model to understanding what this project is (and then how the module and detectors fit in). You never even cover what a verification system is. And that's not really something we can take for granted people will just know.

Please start the tutorial by covering what the project does and how it is useful (not how the module is useful, how the PROJECT is useful). The module is just a tool to be able to build this project.